__tests__/index.spec.js

```diff
@@ -169,6 +169,17 @@ describe('#create', () => {
     expect(userObject).toMatchObject({ email: 'frodo_1@mail.com' });
   });
 
+  it('can handle template attributes', () => {
+    userFabricator = buildTestUserFabricator({
+      name: ObjectFabricator.sequence(n => `TestUser${n}`),
+      email: ObjectFabricator.template(n => `${n.name}@gmail.com`)
```

Comment:
I can think of a couple of solutions but none that are as clear and stable. The best alternative, which I built before but then got rid off because it was less clear and different to the other extensions, is using regex to match for ${...} and to replace it with ${this.filledAttributes....}. 

When filling the attributes I would then have to check if the attribute is a string and then run the regex. 

Do you think that would be better?